crates/sui-graphql-rpc/src/functional_group.rs

```diff
@@ -80,12 +83,16 @@ fn functional_groups() -> &'static BTreeMap<(&'static str, &'static str), Functi
             (("Object", "coins"), G::Coins),
             (("Object", "defaultSuinsName"), G::NameService),
             (("Object", "dynamicField"), G::DynamicFields),
-            (("Object", "dynamicFieldConnection"), G::DynamicFields),
+            (("Object", "dynamicObjectField"), G::DynamicFields),
```

Comment:
ah good catch here

Reply:
The testing support for functional groups is not great (my fault) so it didn't complain before -- we can revisit that post MVP launch.